modules/virt-maintaining-bare-metal-nodes.adoc

```diff
@@ -0,0 +1,11 @@
+// Module included in the following assemblies:
+//
+// virt/node_maintenance/virt-about-node-maintenance.adoc
+
+[id="virt-maintaining-bare-metal-nodes_{context}"]
+= Maintaining bare metal nodes
+
+When you deploy {product-title} on bare metal infrastructure, there are additional considerations that must be taken into account compared to deploying on cloud infrastructure. Unlike in cloud environments where the cluster nodes are considered ephemeral, re-provisioning a bare metal node requires significantly more time and effort for maintenance tasks.
+
+When a bare metal node fails, for example, if the kernel hangs or a NIC card hardware failure occurs, workloads on the failed node need to be restarted elsewhere else on the cluster while the problem node is repaired or replaced. Node maintenance mode allows cluster administrators to gracefully power down nodes, moving workloads to other parts of the cluster and ensuring workloads do not get interrupted. Detailed progress and node status details are provided during maintenance.
```

Comment:
Keeping translation in mind, is there a better word for "hangs"?

Reply:
Good point. I've updated to "if a fatal kernel error happens"